lib/ProductOpener/Import.pm

```diff
@@ -2185,9 +2192,9 @@ sub update_export_status_for_csv_file($) {
 
 	my $args_ref = shift;
 
-	$User_id = $args_ref->{user_id};
-	$Org_id = $args_ref->{org_id};
-	$Owner_id = get_owner_id($User_id, $Org_id, $args_ref->{owner_id});
+	my $User_id = $args_ref->{user_id};
```

Comment:
This may have unwanted side effects.
It's unfortunate, but I put some global variables $User_id, $Owner_id and $Org_id in Display.pm, and some functions use them directly.
So if you make them local, there might be some functions called that uses the global values.

The best thing would be to put those variables in a request structure that we always pass, but that would require changes all over.

Reply:
Hum, I see. I will revert the change document and add a TODO.

Reply:
Could we at least declare them "local" ?